.github/pull_request_template.md

```diff
@@ -6,12 +6,11 @@ Please provide a paragraph or two giving a summary of the change, including rele
 
 - [ ] I have reviewed my diff in github, line by line.
 - [ ] Every change is related to the PR description.
-- [ ] I have [linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) this pull request to the issue(s) that it resolves.
+- [ ] The branch has been merged with/rebased against the head of its merge target.
 - [ ] There are no unexpected formatting changes, superfluous debug logs, or commented-out code.
-- [ ] There are no circuit changes, OR specifications in `/markdown/specs` have been updated.
 - [ ] There are no circuit changes, OR a cryptographer has been assigned for review.
-- [ ] I've updated any terraform that needs updating (e.g. environment variables) for deployment.
-- [ ] The branch has been rebased against the head of its merge target.
-- [ ] I'm happy for the PR to be merged at the reviewer's next convenience.
 - [ ] New functions, classes, etc. have been documented according to the doxygen comment format. Classes and structs must have `@brief` describing the intended functionality.
 - [ ] If existing code has been modified, such documentation has been added or updated.
+- [ ] No superfluous `include` directives have been added.
```

Comment:
There has been an explosion of includes, in large part due to some badly chosen clangd settings. That should be fixed now, and we should make sure we're not lazily adding includes that don't need to be there.